src/data/systeminfo.rs

```diff
@@ -6,7 +6,7 @@ use crate::{PERFORMANCE_DATA, VISUALIZATION_DATA};
 use anyhow::Result;
 use chrono::prelude::*;
 use ctor::ctor;
-use log::{error, trace};
+use log::{trace, warn};
```

Comment:
In general, can we summarize what all are not collected (for which warnings are issued) when requirements are not met and a message is dumped to console or a file/ log?